### DISCUSSION TITLE
Not able to create folders - "Sorry your folder couldn't be saved"

### DISCUSSION THREAD

**Discussion by rajavel1990** [OctoFarm/OctoFarm#1225] [Implemented]
I have installed the latest version 1.6.8 on my windows machine and i am not able to create folders for each of the printers. I get "Sorry your folder couldn't be saved" error.
<img width="2048" alt="Screenshot 2022-09-24 at 8 38 20 PM" src="https://user-images.githubusercontent.com/43663845/192135912-877d716c-f3c2-4aec-809a-f5fb7e8140af.png">

**Comment by NotExpectedYet** [comment]
This is a bug, I've converted it to #1226
